winlogbeat/eventlog/wineventlog.go

```diff
@@ -216,6 +217,11 @@ func (l *winEventLog) buildRecordFromXML(x []byte, recoveredErr error) (Record,
 		e.RenderErr = recoveredErr.Error()
 	}
 
+	if e.Level == "" {
+		//Let's fallback on LevelRaw if the level is not set in the RenderingInfo
```

Comment:
Could you please change this to 

`// Fallback on LevelRaw if the Level is not set in the RenderingInfo.`